packages/core/utils/src/common/normalize-currency-code.ts

```diff
@@ -0,0 +1,12 @@
+import { isString } from "./is-string"
+
+/**
+ * Normalizes `currencyCode` by transforming it to lowercase
+ */
+export function normalizeCurrencyCode(currencyCode: unknown) {
+    if (!isString(currencyCode)) {
+        return
```

Comment:
suggestion: we might need to return the input directly here? also, we always expet a string so I would type the input as string and here if at runtime a non string is provided then we might probably want to throw?